en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mysql.md

```diff
@@ -361,6 +361,9 @@ Follow the  instructions below to set up a MySQL database:
 
 ### Creating the datasource connection to MySQL
 
+!!! note
+    It is recommended to use the UTC zone for database operations as it does not observe daylight savings time (DST). If a database server is hosted in a time zone that is affected by DST, not using UTC could potentially result in an outage (due to the DST shift) when generating OAuth codes or access tokens.
```

Comment:
Shall we change this to something like when `writing to the database` since we cannot write any data to the database with the issue.